client/server/api/index.js

```diff
@@ -20,7 +20,7 @@ export default function api() {
 		response.json( { version } );
 	} );
 
-	if ( config.isEnabled( 'oauth' ) && ! config.isEnabled( 'jetpack-cloud' ) ) {
+	if ( config.isEnabled( 'oauth' ) ) {
 		oauth( app );
 	}
```

Comment:
Enabling the OAuth endpoints on Jetpack cloud. In particular, we care about the `/oauth` endpoint, [which proxies an OAuth login request to the WPCOM API](https://github.com/Automattic/wp-calypso/blob/trunk/client/server/api/oauth.js#L114).